content/integrations/agent_metrics.fr.md

```diff
@@ -66,9 +70,12 @@ Besoin d'aide? Contactez  [l'équipe support de Datadog](http://docs.datadoghq.c
 ## En apprendre plus
 Apprenez en plus sur l'infrastructure monitoring et toutes les intégrations Datadog sur [notre blog](https://www.datadoghq.com/blog/)
 
+<<<<<<< HEAD
```

Comment:
This is a merge conflict artefact. There are numerous examples of this in 9ab59fcee8194c15794a3a3de2055a0f25350798. Definitely need to clean them out.